lib/osf-components/addon/components/gutters/styles.scss

```diff
@@ -85,4 +85,12 @@
 
 .RightGutter {
     @include gutter('right');
+
+    i {
+        color: var(--primary-color);
+
+        &:hover {
+            color: var(--secondary-color);
+        }
+    }
```

Comment:
these style rules don't have anything to do with `.RightGutter` or even the `Gutters` component -- it'd make more sense to put them in your new `Button` component so it looks consistent no matter where it's used